jetbrains-core/src/software/aws/toolkits/jetbrains/ui/connection/AwsConnectionSettingsSelector.kt

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.ui.connection
+
+import com.intellij.openapi.project.Project
+import software.aws.toolkits.core.region.AwsRegion
+import software.aws.toolkits.jetbrains.core.credentials.CredentialManager
+import software.aws.toolkits.jetbrains.core.credentials.ProjectAccountSettingsManager
+import software.aws.toolkits.jetbrains.core.region.AwsRegionProvider
+import javax.swing.JComponent
+
+class AwsConnectionSettingsSelector(
+    private val project: Project,
+    private val settingsChangedListener: (AwsRegion?, String?) -> Unit = { _, _ -> }
```

Comment:
I see it was this way in the original, but I'm surprised that the default value can't just be `{ }`

Reply:
You actually can't, it wants 2 parameters